index.php

```diff
@@ -667,6 +667,8 @@
 							}
 						break;
 						case "editasset":
+							// Get the maximum size of the entry screenshot
```

Comment:
Please avoid comments, as it leads to improper copy-pasting like this :)
Instead, make code readable with clear function and variable names (In this case, the code is fine, just meant "In general", as in, "instead of writing comments")

Reply:
Should have seen that :)

I removed the other unnecessary comment as well.